client/apollo/js/View/Track/AnnotTrack.js

```diff
@@ -664,7 +664,7 @@ define([
                         connectId: featDiv,
                         label: label,
                         position: ["above"],
-                        showDelay: 600
+                        showDelay: 0
```

Comment:
Actually I would recommend `showDelay: 400` because a value of `0` is too quick and gets in the way of mouse-over actions, intentional or otherwise.